tree/readspeed/test/readspeed_general.cxx

```diff
@@ -87,6 +88,11 @@ TEST_F(ReadSpeedIntegration, MultiThread)
 
 TEST_F(ReadSpeedIntegration, NonExistentFile)
 {
+   std::string message = "file ";
+   message += gSystem->pwd();
+   message += "/test_fake.root does not exist";
+   ROOT::TestSupport::CheckDiagsRAII diagRAII{kError, "TFile::TFile", message};
```

Comment:
I think that trying to match the relevant substring instead makes the test simpler and more robust against changes in the absolute path. 
```suggestion
   ROOT::TestSupport::CheckDiagsRAII diagRAII;
   diagRAII.requiredDiag(kError, "TFile::TFile", "/test_fake.root does not exist", /*matchFullMessage=*/false);
```

Reply:
good idea, done :+1: